liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseMultiModuleTest.java

```diff
@@ -63,6 +63,11 @@ protected static void run() throws Exception {
    }
 
    protected static void run(String params, boolean devMode) throws Exception {
+      if (params != null) {
+         params += " -DgenerateFeatures=false"; // turn liberty:generate-features off for multi module tests
+      } else {
```

Comment:
@turkeylurkey we do support generate features for multi-modules don't we? Is this just a test thing?

Reply:
We do support feature generation for multi-module projects. I've turned off `generateFeatures` for the multi-module tests for now as they do not have the sonatype repo in the pom.xml of the test projects and are failing. Re-enabling this can be part of https://github.com/OpenLiberty/ci.maven/issues/1311